File: 2-Basic-Mathematical-Knowledge-for-Algorithms/004-Product_of_3_Integers/c/main.c
#include <stdio.h>

int main() {
    // 1. 入力処理
    int a1, a2, a3;
    scanf("%d %d %d", &a1, &a2, &a3);

    // 2. 積計算処理
    int product = a1 * a2 * a3;
    
    // 3. 出力処理
    printf("%d\n", product);

    return 0;
}

File: 2-Basic-Mathematical-Knowledge-for-Algorithms/004-Product_of_3_Integers/go/main.go
package main

import "fmt"

func main() {
    // 1. 入力処理
    var a1, a2, a3 int
    fmt.Scan(&a1, &a2, &a3)

    // 2. 積計算処理
    product := a1 * a2 * a3

    // 3. 出力処理
    fmt.Println(product)
}